static/app/views/alerts/rules/metric/triggers/index.tsx

```diff
@@ -118,7 +124,13 @@ class Triggers extends Component<Props> {
         <Panel>
           <PanelBody>
             {comparisonType === AlertRuleComparisonType.DYNAMIC ? (
-              <div>{'This is where the anomaly detection alert field choices go'}</div>
+              <AnomalyDetectionForm
```

Comment:
oic
the other components named `Form` as well 😭 
still think we can call this `FormField` though